uportal-war/src/main/webapp/WEB-INF/jsp/BackgroundPreference/viewBackgroundPreference.jsp

```diff
@@ -139,8 +139,8 @@
         </a>
         <c:forEach var="image" items="${thumbnailImages}" varStatus="status">
             <a href="#">
-                <img src="${image}" />
-                <span class="caption">Background ${status.index + 1}</span>
+                <img src="${image}" alt="" role="presentation"/>
```

Comment:
@CR96 -- The title of the PR says "Add alt text," but here it appears the `alt` attribute is blank.  Is there something I'm missing?

Also, I think it would be smart to be prepared for the case that the `backgroundImageCaptions` preference is undefined, which would make the `imageCaptions` array `null` in the JSP.